src/chrome/content/rules/University-of-Exeter.xml

```diff
@@ -26,6 +30,10 @@
 	<securecookie host="^(?:.*\.)?exeter\.ac\.uk$" name=".+" />
 
 
+	<!-- Cert only matches empslocal.ex.ac.uk -->
```

Comment:
No, I'm just preempting the later rewrite so that http://empslocal.ex.ac.uk doesn't turn into https://empslocal.exeter.ac.uk/. Is there a preferred way to do that?